packages-ui/roosterjs-react/lib/emoji/plugin/createEmojiPlugin.ts

```diff
@@ -76,12 +75,11 @@ class EmojiPlugin implements ReactEditorPlugin {
             this.eventHandledOnKeyDown = false;
             if (this.isSuggesting) {
                 this.onKeyDownSuggestingDomEvent(event);
-            } else if (event.rawEvent.which === KeyCodes.backspace && this.canUndoEmoji) {
+            } else if (event.rawEvent.which === KeyCodes.backspace) {
                 //TODO: 1051
                 // If KeyDown is backspace and canUndoEmoji, call editor undo
-                this.editor.undo();
+                this.editor?.undo();
```

Comment:
editor must not be null here. So either check it before the whole block, or add "!"